src/conn/conn_compact.c

```diff
@@ -466,6 +558,7 @@ __compact_server(void *arg)
     WT_STAT_CONN_SET(session, background_compact_running, 0);
 
 err:
+    __background_compact_clear_exclude_entry(session, true);
```

Comment:
```__background_compact_clear_exclude_entry``` could be called ```__background_compact_exclude_list_clear```

Reply:
Done in cc52c025926072ed826202d782fc9641f879cd2a